src/main/java/org/apache/commons/lang3/Validate.java

```diff
@@ -106,7 +108,7 @@ public Validate() {
      */
     public static void isTrue(final boolean expression, final String message, final long value) {
         if (!expression) {
-            throw new IllegalArgumentException(String.format(message, Long.valueOf(value)));
+            throw IllegalArgumentExceptions.format(message, Long.valueOf(value));
```

Comment:
I think the above change shows why the class is unnecessary.
I don't think it makes sense to create a new class just to drop the text 'new ' and 'String.'

Reply:
For this call site, I am 50/50, I implemented here for consistency. Picking this call site to question the existence of the whole PR does not take into account more complex use cases, IMO ;-)

Reply:
The same reasoning applies to all the other places once you have changed the string concatenation to use String.format.